submissions/evgeniy24/html-css-popup-demo/styles/style.scss

```diff
@@ -0,0 +1,379 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Segoe UI", sans-serif;
+  font-size: 13px;
+  color: #686bb1;
+}
+
+body,
+ul,
+ol,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+hr,
+p,
+figure,
+nav,
+button {
+  outline: none;
+  padding: 0;
+  margin: 0;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+/*main menu*/
+.navigation {
+  padding: 10px 10px 0 0;
+}
+
+.navigation__list {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-orient: horizontal;
+  -webkit-box-direction: normal;
+  -ms-flex-direction: row;
+  flex-direction: row;
+  -webkit-box-pack: end;
+  -ms-flex-pack: end;
+  justify-content: flex-end;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+}
+
+.list__item {
+  display: flex;
+  margin-left: 15px;
+  align-items: center;
+  justify-content: center;
+
+  &_bell {
+    width: 38px;
+    height: 38px;
+    border-radius: 50%;
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-pack: center;
+    -ms-flex-pack: center;
+    justify-content: center;
+    -webkit-box-align: center;
+    -ms-flex-align: center;
+    align-items: center;
+  }
+}
+
+.item__apps {
+  background-image: url("../img/icons8-squared-menu-48.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 30px;
+  width: 30px;
+}
+
+.check__input {
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+}
+
+.label__input {
+  width: 38px;
+  height: 38px;
+  border-radius: 50%;
+  display: flex;
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  cursor: pointer;
+}
+
+.item__bell_img {
+  background-image: url("../img/icons8-bell-48.png");
+  background-repeat: no-repeat;
+  background-size: contain;
+  height: 30px;
+  width: 30px;
+}
+
+.item__avatar_img {
+  background-image: url("../img/man.png");
+  background-repeat: no-repeat;
+  background-size: contain;
+  height: 40px;
+  width: 40px;
+}
+
+/*PopUp Content*/
+.popup {
+  display: none;
+  position: absolute;
+  top: 70px;
+  right: 1%;
+  width: 340px;
+  height: auto;
+  max-height: 520px;
+  padding-top: 46px;
+  border: 2px solid #c0c0c0;
+  -webkit-box-shadow: 0 2px 10px rgba(0, 0, 0, 0.2);
+  box-shadow: 0 2px 10px rgba(0, 0, 0, 0.2);
+  overflow-x: hidden;
+}
+
+.popup__menu,
+.popup__menu-continue {
+  margin-left: 25px;
+  margin-right: 25px;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+  -webkit-box-orient: horizontal;
+  -webkit-box-direction: normal;
+  -ms-flex-direction: row;
+  flex-direction: row;
+  -ms-flex-wrap: wrap;
+  flex-wrap: wrap;
+}
+
+.popup__item {
```

Comment:
set the focus for this class, not at element `<a>`

Reply:
Can't understand what do you mean! Please more details

Reply:
All interactive elements should have :focus state.
![popup-focus](https://user-images.githubusercontent.com/26724390/97471067-00b77300-1951-11eb-93cd-c3fe344240a6.png)

Reply:
In the current case, you have only one interactive icon, instead of a full icon block(icon + icon title)
Put block with icon title inside link)

Reply:
I fixed this issue! See the Demo!